src/Sql/dbo/Stored Procedures/OrganizationDomain_Update.sql

```diff
@@ -20,9 +20,9 @@ BEGIN
         [DomainName] = @DomainName,
         [CreationDate] = @CreationDate,
         [VerifiedDate] = @VerifiedDate,
-        [LastCheckedDate] = @LastCheckedDate
+        [LastCheckedDate] = @LastCheckedDate,
```

Comment:
Seems odd that these were let through before?

Reply:
Yeah it's a bit odd, possibly a slightly different rule set. There are also plenty of warnings you get when building it but this was the only error.